### PR TITLE
Fix Joomla 4 FA icon + Joomla 5 name spacing

### PR DESCRIPTION
**Pull request by newlinewebdesign**
- Updated fa icon to work with J4 & J5
- Updated deprecated J classes with name spaced versions
- Updated copyright notice
- Updated more info button with nowrap bs5 class for mobile view

### CODE REVIEW

File: administrator/components/com_j2store/helpers/help.php [hunk 1]
```diff
@@ -2,11 +2,17 @@
 /**
  * @package J2Store
  * @copyright Copyright (c)2014-17 Ramesh Elamathi / J2Store.org
+ * @copyright Copyright (c) 2024 J2Commerce
  * @license GNU GPL v3 or later
  */
 // No direct access to this file
 defined('_JEXEC') or die;
-
+use Joomla\CMS\Factory;
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:10]:
```suggestion
```
declared but not used

File: administrator/components/com_j2store/helpers/help.php [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Uri\Uri;
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:12]:
```suggestion
```

declared but not used

File: administrator/components/com_j2store/helpers/help.php [hunk 3]
```diff
@@ -2,11 +2,17 @@
...
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Session\Session;
+use Joomla\CMS\Version\Version;
```
Comment (brianteeman) [inline administrator/components/com_j2store/helpers/help.php:15]:
```suggestion
```

declared but not used

**Review by newlinewebdesign — COMMENTED** [review]
updated from @brianteeman 's comments

**Comment by brianteeman** [comment]
@newlinewebdesign you dont need to create a new commit when there are suggested changes. You can just click on the button to accept them

image

**Comment by newlinewebdesign** [comment]
@brianteeman thank you. I realized that after I already pushed the update (and still getting my feet wet).  I will get into the proper flow and appreciate all that you have done thus far!

**Pull request merged**